src/ppl/README.md

```diff
@@ -157,68 +95,185 @@ pin placement grid defined with `define_pin_shape_pattern` use:
+
+The `set_simulated_annealing` command defines the parameters for simulated annealing pin placement.
+
+```tcl
+set_simulated_annealing
+    [-temperature temperature]
+    [-max_iterations iter]
+    [-perturb_per_iter perturbs]
+    [-alpha alpha]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-temperature` | Temperature parameter. The default value is `1.0`, and the allowed values are floats `[0, MAX_FLOAT]`. |
+| `-max_iterations` | The maximum number of iterations. The default value is `2000`, and the allowed values are integers `[0, MAX_INT]`. |
+| `-perturb_per_iter` | The number of perturbations per iteration. The default value is `0`, and the allowed values are integers `[0, MAX_INT]`. |
+| `-alpha` | The temperature decay factor. The default value is `0.985`, and the allowed values are floats `(0, 1]`. |
+
+### Place Individual Pin
+
+The `place_pin` command places a specific pin in the specified location with the specified size.
+It is recommended that individual pins be placed before the `place_pins` command,
+as the routing tracks occupied by these individual pins will be blocked, preventing overlaps.
+
+To place an individual pin:
+
+```tcl
+place_pin 
+    [-pin_name pin_name]
+    [-layer layer]
+    [-location {x y}]
+    [-pin_size {width height}]
+    [-force_to_die_boundary]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-pin_name` | The name of a pin of the design. |
+| `-layer` | The routing layer where the pin is placed. |
+| `-location` | The center of the pin (in microns). |
+| `-pin_size` | Tthe width and height of the pin (in microns). |
+| `-force_to_die_boundary` | When this flag is enabled, the pin will be snapped to the nearest routing track, next to the die boundary. |
+
+### Place All Pins
+
+Use the following command to perform pin placement:
+
+```tcl
+place_pins 
+    [-hor_layers h_layers]
+    [-ver_layers v_layers]
+    [-random_seed seed]
+    [-random]
+    [-corner_avoidance length]
+    [-min_distance distance]
+    [-min_distance_in_tracks]
+    [-exclude region]
+    [-group_pins pin_list]
+    [-annealing]
 ```
 
--   The `-hor_multiplier` option defines the thickness multiplier for the horizontal pins.
--   The `-ver_multiplier` option defines the thickness multiplier for the vertical pins.
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-hor_layers` | The layers to create the metal shapes of pins placed in horizontal tracks. It can be a single layer or a list of layer names. |
+| `-ver_layers` | The layers to create the metal shapes of pins placed in vertical tracks. It can be a single layer or a list of layer names. |
+| `-random_seed` | Refer to the developer arguments [here](#developer-arguments). |
+| `-random` | Refer to the developer arguments [here](#developer-arguments). |
+| `-corner_avoidance` | The distance (in microns) from each corner within which pin placement should be avoided. |
```

Comment:
```suggestion
| `-ver_layers` | The layers to create the metal shapes of pins placed in vertical tracks. It can be a single layer or a list of layer names. |
| `-corner_avoidance` | The distance (in microns) from each corner within which pin placement should be avoided. |
```